fetchArtistGenre.py

```
@@ -0,0 +1,6 @@
+import spotipy
+from spotipy.oauth2 import SpotifyClientCredentials
+
+sp = spotipy.Spotify(auth_manager=SpotifyClientCredentials(client_id='088effd1d43c44c1b859616babe82da6', client_secret='dab4db21446140489cbc75ab7ec7ed58'))
```

Ruff (0.14.5)

Possible hardcoded password assigned to argument: "client_secret"

(S106)